client/webserver/webserver.go

```diff
@@ -627,6 +632,39 @@ func New(cfg *Config) (*WebServer, error) {
 	return s, nil
 }
 
+// fetchLatestVersion is a helper function to retrieve the latest version of the app
+// from github.
+func (w *WebServer) fetchLatestVersion(ctx context.Context) {
+	var response struct {
+		TagName string `json:"tag_name"`
+	}
+
+	err := dexnet.Get(ctx, "https://api.github.com/repos/decred/dcrdex/releases/latest", &response, dexnet.WithSizeLimit(1<<22))
+	if err != nil {
+		log.Debugf("Error getting latest version: %v", err)
+		return
+	}
+
+	lastestVersion := latestVersionRegex.FindString(response.TagName)
+	lastestMajor, latestMinor, latestPatch, _, _, err := version.ParseSemVer(lastestVersion)
+	if err != nil {
+		log.Debugf("Error parsing latest version: %v", err)
+		return
+	}
+
+	currentMajor, currentMinor, currentPatch, _, _, err := version.ParseSemVer(w.appVersion)
+	if err != nil {
+		log.Debugf("Error parsing app version: %v", err)
+		return
+	}
+
+	if lastestMajor > currentMajor ||
+		(lastestMajor == currentMajor && latestMinor > currentMinor) ||
+		(latestMinor == currentMajor && latestMinor == currentMinor && latestPatch > currentPatch) {
+		w.newAppVersionAvailable = true
+	}
```

Comment:
`latestMinor == currentMajor` cant be right.

Are you sure what I said isnt right?

Reply:
I see, if local major or minor would be less, that's an automatic upgrade.